tasks/add_new_openwrap_partner.py

```diff
@@ -1127,81 +1279,88 @@ def main():
   elif setup_type == constant.ADPOD:
       roadblock_type = 'ONE_OR_MORE'
 
-  logger.info(
-    u"""
-
-    Going to create {name_start_format}{num_line_items}{format_end} new line items.
-      {name_start_format}Order{format_end}: {value_start_format}{order_name}{format_end}
-      {name_start_format}Advertiser{format_end}: {value_start_format}{advertiser}{format_end}
-      {name_start_format}Advertiser Type{format_end}: {value_start_format}{advertiser_type}{format_end}
-      {name_start_format}LineItem Type{format_end}: {value_start_format}{lineitem_type}{format_end}
-      {name_start_format}LineItem Prefix{format_end}: {value_start_format}{lineitem_prefix}{format_end}
-      {name_start_format}Setup Type{format_end} = {value_start_format}{setup_type}{format_end}
-      {name_start_format}Use 1x1 Creative{format_end} = {value_start_format}{use_1x1}{format_end}
-        
-    Line items will have targeting:
-      {name_start_format}rates{format_end} = {value_start_format}{prices_summary}{format_end}
-      {name_start_format}bidders{format_end} = {value_start_format}{bidder_code}{format_end}
-      {name_start_format}placements{format_end} = {value_start_format}{placements}{format_end}
-      {name_start_format}custom targeting{format_end} = {value_start_format}{custom_targeting}{format_end}
-      {name_start_format}same advertiser exception{format_end} = {value_start_format}{same_adv_exception}{format_end}
-      {name_start_format}device categories{format_end} = {value_start_format}{device_categories}{format_end}
-      {name_start_format}device capabilities{format_end} = {value_start_format}{device_capabilities}{format_end}
-      {name_start_format}roadblock type{format_end} = {value_start_format}{roadblock_type}{format_end}
-    """.format(
-      num_line_items = len(prices),
-      order_name=order_name,
-      advertiser=advertiser_name,
-      advertiser_type=advertiser_type,
-      lineitem_type=lineitem_type,
-      lineitem_prefix=lineitem_prefix,
-      setup_type=setup_type,
-      user_email=user_email,
-      prices_summary=prices_summary,
-      bidder_code=bidder_code,
-      placements=placements_print,
-      sizes=sizes,
-      custom_targeting=custom_targeting,
-      same_adv_exception=same_adv_exception,
-      device_categories=device_categories,
-      device_capabilities=device_capabilities,
-      roadblock_type=roadblock_type,
-      use_1x1=use_1x1,
-      name_start_format=color.BOLD,
-      format_end=color.END,
-      value_start_format=color.BLUE,
-    ))
 
-  if setup_type == constant.ADPOD:
+  if deal_lineitem_enabled == False: 
```

Comment:
Can we make it non adpod for better understanding: 
`if setup_type is not constant.ADPOD:`

Reply:
We are printing this summary  for Non Adpod and Adpod (non deal) line items.

Reply:
I was referring to the comment on Line no 1284: 
`# Prininting non adpod lineitem summary`